src/components/gravid/validateGravid.ts

```diff
@@ -10,6 +10,7 @@ import { pushFeilmelding } from '../felles/Feilmeldingspanel/pushFeilmelding';
 import { Omplassering } from './Omplassering';
 import { i18n } from 'i18next';
 import LangKey from '../../locale/LangKey';
+import { validateTermindato } from '../../validation/validateTermindato';
```

Comment:
Denne kan være default eksportert for å være mere lesbar.

export default validateTermindato